woocommerce/wc-admin
# Tweak: change MailPoet's title and description in Store Profiler (#6989)

Fixes #6989

Change title and description of MailPoet extension in Store Profiler.

### Screenshots

<img width="743" alt="screenshot-2021-04-30-at-20 22 10" src="https://user-images.githubusercontent.com/470616/118249090-51358e80-b4a5-11eb-994e-0f95e1592388.png">

### Detailed test instructions:

- Create a brand new site and go to the OBW.
- In the first OBW step (Store Details), set a Country / Region other than the US or other large markets (e.g., Uruguay).
- Continue with the profiler.
- In the 4th step (Business Details), choose any of the options in both selectors.
- Verify that the MailPoet extension has a new title and description
